packages/react/src/themes/teams/components/Loader/loaderStyles.ts

```diff
@@ -48,12 +45,9 @@ export default {
     }
     const svgAnimation: ICSSInJSStyle = {
       animationName: {
-        keyframe: () =>
-          ({
-            to: {
-              transform: `translate3d(0, ${v.svgTranslatePosition[p.size]}, 0)`,
-            },
-          } as any),
+        to: {
```

Comment:
I like that we have the two options when defining animations - directly defining keyframes, as well as `keyframe` + `params`.